README.md

```diff
@@ -16,6 +16,7 @@ The `main.tf` file will configure a Kubernetes Secret and DaemonSet which will t
 |------|-------------|------|---------|
 | lacework_access_token | The Lacework Access Token for the Datacollector to use | `string` | null |
 | lacework_agent_name | The name for the Lacework agent service within Kubernetes | `string` | "lacework-agent" |
+| lacework_agent_interface_connection_size | Desired value for the InterfaceConnectionSize Lacework agent parameter | `string` | "10000" |
```

Comment:
This new parameter is not documented: https://support.lacework.com/hc/en-us/articles/360024365753-Configure-Agent-Behavior-in-config-json-File

We should write a documentation and place a link to the documentation in the module readme.